src/screens/home.js

```diff
@@ -40,30 +109,30 @@ const HomeScreen = ({ navigation }) => {
                     <View style={styles.dividingLineContainer} />
                     <View style={styles.amountContainer}>
                         <Text style={{ fontSize: 14, color: '#91919F', alignSelf: 'center' }}>Account Balance</Text>
-                        <Text style={{ fontSize: 40, color: '#black', fontWeight: 'bold', marginTop: 20 }}>9400.0</Text>
+                        <Text style={{ fontSize: 40, color: '#black', fontWeight: 'bold', marginTop: 20 }}>{accountBalance}</Text>
                     </View>
                     <View style={styles.buttonContainer}>
                         <View style={styles.incomeContainer}>
                             <View style={styles.iconContainer}>
                                 <Image source={require('../assets/icons/arrow-down.png')} style={{ height: 10, width: 11 }} />
                                 <Image source={require('../assets/icons/income-sq.png')} style={{ height: 16, width: 24 }} />
                             </View>
-                            <Text style={{ fontSize: 17, color: 'white' }}>Income{'\n'}25000</Text>
+                            <Text style={{ fontSize: 17, color: 'white' }}>Income{'\n'}{totalIncome}</Text>
                         </View>
                         <View style={styles.expensesContainer}>
                             <View style={styles.iconContainer}>
                                 <Image source={require('../assets/icons/arrow-up.png')} style={{ height: 10, width: 11 }} />
                                 <Image source={require('../assets/icons/expense-sq.png')} style={{ height: 16, width: 24 }} />
```

Comment:
Please move all image imports to a single file.

images.js
eg: const expense = require("path to image")

in component -
import {expense} from 'path_image_file'